stories/FileTreeNode.stories.tsx

```diff
@@ -23,48 +28,47 @@ const getProps = ({
     isFolder,
     level,
     node: {
-      id: 'node-id',
+      id: nodeId,
       name: nodeName,
     },
     renderChildNodes,
     onSelect: () => {},
   };
 };
 
+const render = ({ versionId = 123, store = configureStore(), ...props }) => {
+  return renderWithStoreAndRouter(
```

Comment:
The `FileTree` stories also need to be wrapped in a store (depending on how you address previous comments). Right now they are broken for me with a red screen. Hmm. The tests should have broken for that; I'll file an issue.